main.go

```diff
@@ -557,7 +558,27 @@ The commands work as following:
 
 	b, _ = arguments.Bool("info")
 	if b {
-		printDeviceInfo(device)
+		if display, _ := arguments.Bool("display"); display {
+			if address == "" || rsdPort == 0 {
+				log.Fatal("Please provide --address and --rsd-port to retreive display information")
+				return
+			}
+
+			deviceInfo, err := deviceinfo.NewDeviceInfo(device)
+			exitIfError("Can't connect to deviceinfo service", err)
+			defer deviceInfo.Close()
+
+			info, err := deviceInfo.GetDisplayInfo()
+			exitIfError("Can't fetch dispaly info", err)
+
+			log.WithField("display", info).Info("Got display info")
```

Comment:
```suggestion
			fmt.Println(convertToJSONString(info))
```
to have the same output format